week-1-ts-ecosystem/w01e01-typescript-basics/package.json

```diff
@@ -1,5 +1,11 @@
 {
   "name": "w01e01-typescript-basics",
+  "author": "Dariusz Gala",
   "version": "1.0.0",
-  "scripts": {}
+  "scripts": {
+    "start": "npx tsx src/*"
```

Comment:
Jeśli korzystasz w projekcie z narzędzia `tsx` to warto je zainstalować i skonfigurować odpowiednio skrypty.

To znaczy rozważ żeby:
[tsx](https://www.npmjs.com/package/tsx)

nie było uruchamianę komendą `npx` tylko zainstalowane jako `devDependency` 

Dodaj również skrypt do tzw. [watch mode](https://www.npmjs.com/package/tsx#user-content-watch-mode)
wtedy będziemy mieli skypt do build i do dev

Brakuje mi też w projekcie `tsconfig.json`
Warto go dodać komendą `tsc --init` puszczaną z npm run (np. wpisać to sobie na chwilę jako `script` i uruchomić)
albo tak jak teraz  robisz, to uruchomić jednorazowo: `npx tsc --init`